docs/user-guide/overview.md

```diff
@@ -33,7 +33,9 @@ Documentation for other releases can be found at
 
 # Kubernetes Overview
 
-Kubernetes is an open-source system for managing containerized applications across multiple hosts in a cluster. It provides mechanisms for application deployment, scheduling, updating, maintenance, and scaling. A key feature of Kubernetes is that it actively manages the containers to ensure that the state of the cluster continually matches the user's intentions.
+Kubernetes is an open-source system for managing containerized applications across multiple hosts in a cluster. For applications / micro-services that run on Kubernetes, we want deployments to be easy but powerful. 
```

Comment:
I'm not sure it's the deployments you want to be easy and powerful. Maybe "Kubernetes is intended to make deploying containerized/microservice-based applications easy but powerful."